scripts/gen-nncp.sh

```diff
@@ -141,6 +141,7 @@ EOF_CAT
       - destination: 0.0.0.0/0
```

Comment:
So, does this need to stay `0.0.0.0/0`? We can't just use the ctlplane subnet for it?

Reply:
For now yes, when trying to use the ctlplane subnet nmstate operator fails to apply the config sometimes due to an unrelated IPV6 route test ping.

I left some details on that in the Jira but I would like to resolve that in a follow up patch to unblock 4.14 for now.

Reply:
For some context, when nmstate operator applies the policy it then generates a list of probes to test. This comes from the generated output of `nmstatectl show` and tries to ping each gateway. If one of the pings fail the policy is rolled back and failed, I need to touch base with the nmstate folks.